<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup the game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QComboBox" name="numberRounds">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the number of rounds to win a game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="player1Name"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="gameType"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="player2Name"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Rounds </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Player &amp;1 name </string>
     </property>
     <property name="buddy">
      <cstring>player1Name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Player &amp;2 name</string>
     </property>
     <property name="buddy">
      <cstring>player2Name</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Game type </string>
     </property>
     <property name="buddy">
      <cstring>gameType</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="sizeBoard">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the size of the board (n x n)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Board size </string>
     </property>
     <property name="buddy">
      <cstring>sizeBoard</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player1Name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateOkButtonState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player2Name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateOkButtonState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gameType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateBoardSizeSpinBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateOkButtonState()</slot>
  <slot>updateBoardSizeSpinBox()</slot>
 </slots>
</ui>
